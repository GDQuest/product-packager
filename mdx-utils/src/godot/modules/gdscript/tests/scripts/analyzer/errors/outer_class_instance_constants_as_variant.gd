class Outer:
	const OUTER_CONST: = 0
	class Inner:
		pass

func test() -> void:
	var instance: = Outer.Inner.new()
	var instance_v: Variant = instance
	print(instance_v.OUTER_CONST)
