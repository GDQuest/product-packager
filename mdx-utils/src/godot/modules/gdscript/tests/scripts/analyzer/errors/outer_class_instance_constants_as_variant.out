GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR
>> on function: test()
>> analyzer/errors/outer_class_instance_constants_as_variant.gd
>> 9
>> Invalid get index 'OUTER_CONST' (on base: 'RefCounted (Inner)').
